sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample2_RecognizeReceiptsFromFile.cs

```diff
@@ -24,7 +24,7 @@ public async Task RecognizeReceiptsFromFile()
             FormRecognizerClient client = new FormRecognizerClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
             #endregion
 
-            string receiptPath = FormRecognizerTestEnvironment.JpgReceiptPath;
+            string receiptPath = FormRecognizerTestEnvironment.CreatePath("contoso-receipt.jpg");
```

Comment:
I realized some samples were using the `FormRecognizerTestEnvironment.SomethingPath` properties, while others were creating the path directly from `CreatePath`. I'm keeping them consistent and trying to make them as clear as possible.

1. I think it's easier for users to understand what's going on if we use the actual filenames in samples instead of `TestFilename.Something`.
2. `CreatePath` and `CreateUriString` are preferred in samples to `CreateStream` and `CreateUri`. Users in general will have a string (path or URI) and using the later methods would hide how we're handling them.